windows/src/engine/kmcomapi/util/utilhandleexception.pas

```diff
@@ -108,50 +102,12 @@ procedure LogException(const SourceClassName: string; E: Exception; ExceptAddr:
     ;
   end;
 {$ELSE}
-var
-  errlog: TStrings;
-  msg, errlogfile: string;
 begin
-  try
-    if E = nil
-      then msg := Format('Exception in %s at %p', [SourceClassName, ConvertedExceptAddr(ExceptAddr)])
-      else msg := Format('Exception in %s at %p (%s): %s', [SourceClassName, ConvertedExceptAddr(ExceptAddr), E.ClassName, (E as Exception).Message]);
-    KL.LogError(msg);
-
-    errlog := TStringList.Create;
-    try
-      errlogfile := GetErrLogFileName('kmcomapi');  // I2824
-
-      JclLastExceptStackListToStrings(errlog, True, True, True, False);
-
-      KL.LogError(errlog.Text);
-
-      errlog.Text :=
-        'Crash Identifier: kmcomapi.dll_'+GetVersionString+'_'+IntToHex(Integer(ExceptAddr),8)+#13#10#13#10+
-        'KMCOMAPI EXCEPTION AT '+FormatDateTime('yyyy-mm-dd hh:nn:ss', Now) + #13#10 +
-        'kmcomapi.dll version ' + GetVersionString + #13#10 +
-        msg + #13#10#13#10 +
-        errlog.Text + #13#10#13#10;
-
-      if FileExists(errlogfile) then
-        with TStringList.Create do
-        try
-          LoadFromFile(errlogfile);  // use prolog encoding
-          errlog.Text := Text + errlog.Text;
-        finally
-          Free;
-        end;
-      errlog.SaveToFile(errlogfile, TEncoding.UTF8);  // I3337
-    finally
-      errlog.Free;
-    end;
-  except
-    ;
-  end;
+  {$MESSAGE HINT 'TODO: Write a raw call stack to diag folder which can then be sucked in by client app and reported'}
```

Comment:
I assume this means a temporary loss in functionality?

Reply:
Yeah, a little bit -- these call stacks never were particularly accessible and so were little used in diagnostics. Hence I am not too worried about leaving them out for now.